src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -332,7 +335,14 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
     }
     if (maxSlope?.valid) scenarioConfig.max_slope = parseFloat(maxSlope.value);
     if (selectedQuestion?.valid) {
-      scenarioConfig.priorities = selectedQuestion.value['priorities'];
+      scenarioConfig.scenario_priorities =
```

Comment:
Im wondering if it makes sense to have this done at a service layer, so we can have whatever shape we need to work with on the backend/forsys and then a consistent shape on the FE so we don't need to do mapping whenever we consume this data. 
Thinking that it would be great if we can have a simple and repeatable way of adding questions to the scenario config. My concern is that we don't really have any shape or interface to understand how the two things are wired together, and additionally, seems like we are combining camelCase and snake_case for properties which makes is a bit harder to use in the future

Reply:
I can move it to the service! And then change scenarioConfig to save a TreatmentGoalQuestion. The camelCase/snake_case issue is one I'm fixing now